app/migrations/seed.py

```diff
@@ -0,0 +1,604 @@
+"""
```

Comment:
So if we're not bringing across new GE tokens, we won't need this right?

Reply:
just not the community token part of it, we still need the USSD menu part